utils/graphene/context.py

```diff
@@ -0,0 +1,10 @@
+from django.utils.functional import cached_property
+
+
+class GQLContext:
+    def __init__(self, request):
+        self.request = request
+
+    @cached_property
```

Comment:
We could probably have just used `property` here. I don't see anything to actually "cache" here.